### PR TITLE
Add xz blog

### PR DESCRIPTION
**Pull request by abnegate**
## What does this PR do?

Add blog about xz backdoor 

## Test Plan

(Write your test plan here. If you changed any code, please provide us with clear instructions on how you verified your changes work.)

## Related PRs and Issues

(If this PR is related to any other PR or resolves any issue or related to any issue link all related PR and issues here.)

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

(Write your answer here.)

### CODE REVIEW

File: src/routes/blog/post/ensuring-security-amidst-xz-concern/+page.markdoc [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+author: jake-barnby
+category: engineering
+featured: false
+---
+
+In the light of recent unsettling revelations regarding a backdoor discovered in the widely-used xz utility, a compression tool used in Linux environments, including Red Hat and Debian systems, the cyber-security landscape has been abuzz with concern. This discovery had a large potential impact on encrypted SSH connections, a backbone of secure communications in the tech world.
+
+At Appwrite, ensuring the security and trust of our developers and users is paramount. We understand the concerns that arise from such vulnerabilities and their potential implications. It's crucial for our community to know that Appwrite's services remain unaffected by the xz backdoor. This issue affected beta and test versions of Red Hat and Debian distributions, which Appwrite does not use.
+
+# What Does This Mean for Self-Hosted Appwrite Solutions?
```
Comment (eldadfux) [inline src/routes/blog/post/ensuring-security-amidst-xz-concern/+page.markdoc:17]: We need to update all title cases.

Comment (eldadfux) [reply src/routes/blog/post/ensuring-security-amidst-xz-concern/+page.markdoc:17]: Title should be h2

File: src/routes/blog/post/ensuring-security-amidst-xz-concern/+page.markdoc [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+This incident serves as a reminder of the challenges and resources required to maintain a secure self-hosted environment. Appwrite's cloud solutions offer a simpler alternative, where the burden of security and infrastructure management is lifted off your shoulders. Our team ensures that your data remains secure, and your services uninterrupted, allowing you to focus on what you do best - creating amazing applications.
+
+In a world where cyber threats are evolving at an alarming pace, choosing Appwrite's cloud solutions is more than a convenience; it's a strategic decision towards ensuring the security and reliability of your applications.
+
+For any further queries or concerns, feel free to reach out. We're here to support you every step of the way.
+
+# Resources
+
+Visit our documentation to learn more about Appwrite, join us on Discord to be part of the discussion, view our blog and YouTube channel, or visit our GitHub repository to see our open-source code.
```
Comment (eldadfux) [inline src/routes/blog/post/ensuring-security-amidst-xz-concern/+page.markdoc:37]: We can skip this section.

**Review by eldadfux — CHANGES_REQUESTED** [review]

File: src/routes/blog/post/ensuring-security-amidst-xz-concern/+page.markdoc (hunk 1 above)
Comment (gewenyu99) [reply src/routes/blog/post/ensuring-security-amidst-xz-concern/+page.markdoc:17]: The markdown pre-processor turns all headings 1 level lower, this is rendered as H2

**Review by gewenyu99 — APPROVED** [review]

**Pull request merged**